app/models/spree/calculator/weight/package_builder.rb

```diff
@@ -132,11 +132,17 @@ def packages(package)
 
       def get_max_weight(package)
         order = package.order
-        max_weight = max_weight_for_country(order.ship_address.country)
-        if max_weight == 0 && max_weight_per_package > 0
-          max_weight = max_weight_per_package
+        order_ship_address_country = order.ship_address.country
```

Comment:
I am not sure if there is any benefit of pulling this out into a local, it is only referenced in one place in this method.